ui/app/components/job-page/parts/recent-allocations.js

```diff
@@ -3,6 +3,8 @@ import { computed } from '@ember/object';
 import PromiseArray from 'nomad-ui/utils/classes/promise-array';
 
 export default Component.extend({
+  classNames: ['boxed-section'],
```

Comment:
Guessing this might have made it in here by mistake?

Reply:
Ah ignore this, I've just seen where you are using `.boxed-section` further down? Actually if this class is just for testing is there a more appropriate way of doing it? Maybe not, but thought I'd ask. I'm guessing you are running this as as a production build and therefore can't use the `data-test-* attributes?

Reply:
Nah, this class was added to fix a bug where two elements on the parameterized-child job page had no margin between them.

That it's used in the puppeteer script is semi-coincidence and semi-sadness. As you suspected, I'm not using the `data-test-*` attributes because they aren't available. Not because it's a production build but because `data-test-*` is only in test builds 😢 It might be worth configuring that to show up in dev builds.

Reply:
Oh weird yeah, 'pretty sure' they show up in our dev builds, though I might be misremembering, I probably have a slightly different config to nomad though. What would be ace is if you could use PageObjects here. 

This is one of those classic things that changes without people realising though, if it could share the PageObjects/`data-test-*` things that would be great.

Just bouncing this around for the chat more than anything though, I agree with your goal with this 🔨

Reply:
Using the same PageObjects would be super awesome. Granted the ember bindings would have to be replaced with puppeteer ones...and somehow shared across the UI code base and this script...but I can see it happening and being much better than what I'm doing right now 🙈